README.md

```diff
@@ -19,6 +21,21 @@
 [bluestyle-img]: https://img.shields.io/badge/code%20style-blue-4495d1.svg "Blue Style"
 [bluestyle-url]: https://github.com/invenia/BlueStyle
 
-CompatHelper is a Julia package that helps you keep your `[compat]` entries up-to-date.
+## Setup
+### GitHub
+TODO
+
+### GitLab
+TODO
+### Environment Variables
 
-Please see the [documentation](https://JuliaRegistries.github.io/CompatHelper.jl/dev/).
+| Name | Description |
+| ---- | ----------- |
+| CI_PROJECT_PATH | Name on GitLab of the `organization/repo`, provided by default on GitLab |
+| GIT_COMMITTER_NAME | Name to associate commits with |
+| GIT_COMMITTER_EMAIL | Email to associate commits with |
+| GITHUB_ACTOR | Users on GitHub to CC on pull requests |
+| GITHUB_REPOSITORY | Name on GitHub of the `organization/repo`, provided by default on GitHub |
+| GITHUB_TOKEN | GitHub Access Token, provided by default by GitHub Actions |
+| GITLAB_TOKEN | GitLab Access Token, provided by default by GitLab CI |
```

Comment:
I don't think either of these are provided by default? I thought these were the ones we need to make secrets for.

Reply:
https://cli.github.com/manual/gh_help_environment

I am unsure of GL here. We needed to definitely provide that since we're launching off of GHA.

Reply:
hmm right. I guess it kind of depends on the permissions you have set for that user. 

GitLab has similar tokens such as `CI_JOB_TOKEN` (https://docs.gitlab.com/ee/ci/variables/predefined_variables.html) which creates a token for that particular job, but I don't think you can access the `master` Personal Access Key for the user running the pipeline/job.

Looks like this is what the job token gets you (https://docs.gitlab.com/ee/api/index.html#gitlab-cicd-job-token) which I don't think would be sufficient for making MR's to the project. I think for GL you might have to specify the access token in the secrets rather than relying on this. 

I might try some tests using the CI_JOB_TOKEN to see if that would work if you ran CH on GitLab in a similar way that we use it on GitHub.

Reply:
Also looking at the page you linked, I don't think those are provided by GitHub Actions, as at the top it says `Environment variables that can be used with gh`

Reply:
Here is the list of GitHub Pre-Defined environment variables https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables

Reply:
Hmmm I can't seem to find it anymore, maybe I'm just blind. There's a default ENV VAR which provides the `GITHUB_TOKEN`, or something like that for making authenticated requests to the GH API so you don't get rate limited.

Reply:
Even in the integration tests we need to make `GITHUB_TOKEN` specifically from the secret https://github.com/JuliaRegistries/CompatHelper.jl/blob/master/test/integration-tests.jl#L38, and in the unit tests we just mock all of the actual GitForge API calls, so not too sure which token or env var you might be thinking of.